src/genie/libs/parser/viptela/show_control_connections.py

```diff
@@ -0,0 +1,91 @@
+# Metaparser
```

Comment:
maybe logic between iosxe and viptela are identical? if so, actually we can inherit the class from another and reduce redundant code.

Reply:
They may be, but I've found so much nuance between viptela & iosxe, it's been simpler to keep things separate. I'll defer to the pyats team on this. I'm good either way

Reply:
sure. once your changes are done, we will update.